tests/unit/test_sonata/test_split_population.py

```diff
@@ -533,8 +533,8 @@ def _check_biophysical_nodes(path, has_virtual, has_external):
             expected_mapping["V2"] = {"old_id": [0], "new_id": [0]}
 
         if has_external:
-            expected_mapping["external_A__B"] = {"old_id": [5], "new_id": [0]}
-            expected_mapping["external_A__C"] = {"old_id": [5], "new_id": [0]}
+            expected_mapping["external_A_B"] = {"old_id": [5], "new_id": [0]}
+            expected_mapping["external_A_C"] = {"old_id": [5], "new_id": [0]}
```

Comment:
Same here, this should be only one external node population I believe:
```suggestion
            expected_mapping["external_A"] = {"old_id": [5], "new_id": [0]}
```